### DISCUSSION TITLE
使用Docker打造自己的云平台

### DISCUSSION THREAD

**Discussion by ycrxun** [ycrxun/ycrxun.github.io#25] [Announcements]
https://ronggle.com/2018/cloud-labs/ 

 失踪人口回归，经过几个月的失踪，我又终于回归正轨了，前面没有更博客，主要原因是，因为我实在是太忙（懒）了。

**Comment by looser098** [comment]
苍老师，我看不懂

**Comment by ycrxun** [comment]
> @looser098
> 苍老师，我看不懂

哪里不懂？这个主要是用来记录我自己研究的一些关键步骤的笔记，如果你想深入了解，可以进一步给你解答。

**Comment by looser098** [comment]
我只是一句玩笑话，到你的这个境界我还差的很远，编程，你是专业的。我只是进来偷窥一下!哥们，你不知道我是谁吧？现在在哪里工作？

**Comment by ycrxun** [comment]
> @looser098
> 我只是一句玩笑话，到你的这个境界我还差的很远，编程，你是专业的。我只是进来偷窥一下!哥们，你不知道我是谁吧？现在在哪里工作？

抱歉，我没看出来你是谁，我没啥境界，就瞎鼓捣，图一时乐趣，哈哈😄

**Comment by dingchenghong** [comment]
博主，有个问题想问下，容器要使用traefik的网络，那么是否说明，容器必须要和traefik在同一台主机了，容器和traefik分主机部署是否可行呢？貌似不可行哦，这么说的话！

**Comment by ycrxun** [comment]
@dingchenghong 如果想使用直接使用traefik来暴露容器，那么就必须保证traefik和而需要暴露的容器在同一个网络（docker network），而我文中使用的是docker swarm模式，我只需要保证他们在一个网络里，至于他们在哪一个机器上，我并不关心。
我理解的您这边说的`容器和traefik分主机部署`，可能不能直接将容器暴露出来，但是可以通过其他方式完成。
